pyaqueduct/api.py

```diff
@@ -76,20 +88,37 @@ def get_experiment(self, alias: str) -> Experiment:
             created_at=experiment_data.created_at,
         )
 
-    def find_experiments(self, search: str, limit: int, offset: int = 0) -> List[Experiment]:
-        """Find the experiments that have the search criteria in their title or id.
+    @validate_call
+    def find_experiments(
```

Comment:
Just a kind request to make sure that somewhere in the call stack (probably on the server side) ordering is preserved. You have pagination here, so probably results should be consistent from call to call.